src/main/java/com/openshift/restclient/model/IHandler.java

```diff
@@ -0,0 +1,27 @@
+package com.openshift.restclient.model;
+
+import com.openshift.internal.restclient.model.JSONSerializeable;
+
+import java.util.Optional;
+
+/**
+ * @author Ulf Lilleengen
+ */
+public interface IHandler extends JSONSerializeable {
+    enum HandlerType {
```

Comment:
We have made a deliberate decision to avoid proper enums  in order to make the library lenient to changes.  Please convert to const.